debian/changelog

```diff
@@ -1,3 +1,9 @@
+piserial (2.0.2-2) buster; urgency=medium
+
+  * Remove the Depends of net-tools
```

Comment:
Recreate the changelog with the " Replace depend of net-tools with iproute2 " commit.